xcode/Subconscious/Shared/Components/StackedGlowingImage.swift

```diff
@@ -8,37 +8,36 @@
 import Foundation
 import SwiftUI
 
-struct StackedGlowingImage: View {
+struct StackedGlowingImage<Content: View>: View {
```

Comment:
This effect works on any view now, which I needed for the `GenerativeProfilePicture`